cgi-bin/CATS/Problem/Storage.pm

```diff
@@ -159,6 +159,14 @@ sub load {
     $self->load_problem(@rest);
 }
 
+sub _is_file_exist {
```

Comment:
1) Should this be in Repository class?
2) We have working copy, perhaps just check file existence? Alternatively, use `ls-files`?